lib/Github/Api/CurrentUser/Memberships.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Github\Api\CurrentUser;
+
+use Github\Api\AbstractApi;
+
+class Memberships extends AbstractApi
+{
+    /**
+     * List your organization memberships.
+     *
+     * @link https://developer.github.com/v3/orgs/members/#get-your-organization-membership
+     *
+     * @return array
+     */
+    public function all()
+    {
+        return $this->get('user/memberships/orgs');
+    }
+
+    /**
+     * Get your organization membership.
+     *
+     * @link https://developer.github.com/v3/orgs/members/#get-your-organization-membership
+     *
+     * @return array
+     */
+    public function organization($organization)
+    {
+        return $this->get('user/memberships/orgs/'.rawurlencode($organization));
+    }
+
+    /**
+     * Edit your organization membership
+     *
+     * @link https://developer.github.com/v3/orgs/members/#edit-your-organization-membership
+     *
+     * @return array
+     */
+    public function edit($organization)
```

Comment:
the name is weird, because this method only allows to mark the membership as active.